src/components/networked.js

```diff
@@ -6,7 +6,7 @@ var bind = AFRAME.utils.bind;
 AFRAME.registerComponent('networked', {
   schema: {
     template: {default: ''},
-    showLocalTemplate: {default: true},
+    attachLocalTemplate: { default: true },
```

Comment:
This might be an annoying change at this point, so I'll leave it to your discretion. When I read `attachLocalTemplate`, it implies the concept of a "local template". Maybe this would be better named `attachTemplateLocally` or `attachTemplateToLocal` or `attachTemplateToLocalEntity`.